dds/DCPS/QOS_XML_Handler/README.md

```diff
@@ -0,0 +1,9 @@
+# QoS XML Handler
+
+Optional utility to allow one to define the QoS of DDS unities using XML.
```

Comment:
```suggestion
Optional utility to allow one to define the QoS of DDS enities using XML.
```

Reply:
I think I hit the wrong suggestion in the spell check.